### PR TITLE
First version of boilerplate files

### PR DESCRIPTION
**Pull request by gmarek**
cc @timothysc @wojtek-t

### CODE REVIEW

**Comment by gmarek** [comment]
@thockin PTAL if the boilerplate LG.

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Perf-tests release process
+
+Performance tests are released around main Kubernetes release dates, to keep tests which given release went through. The process is as follows:
+
+1. Kubernetes minor release vX.X is published
+1. Issue for cutting release vX.X for perf tests is created
+1. All [OWNERS](OWNERS) lgtm the release
+1. An OWNER creates the release branch from the master
+1. Ann announcement email is sent to `kubernetes-dev@googlegroups.com`
```
Comment (timothysc) [inline RELEASE.md]: Shouldn't we also cross post to some other list.  perhaps scale?

**Comment by timothysc** [comment]
Generally lgtm but I don't have contributor rights to this repo.  @gmarek

File: RELEASE.md (hunk 1 above)
Comment (gmarek) [inline RELEASE.md]: I'm not a huge fan of duplicating communication channels. This probably won't be of a huge interest to non-devs, as cutting a release is mostly a tool for "freezing" tests for a given release for future reference/regression testing for cherry-picks. I guess most of the people would be interested in tools from head mostly.

**Comment by gmarek** [comment]
@thockin - can you give @timothysc access to this repo?

**Comment by eparis** [comment]
I just looked and can't figure out how @gmarek has permission on this repo...

**Comment by eparis** [comment]
do you have merge rights?

**Comment by eparis** [comment]
@thockin I created https://github.com/orgs/kubernetes/teams/perf-tests-maintainers and added @timothysc 
Both Tims can now add/approve people with write permissions to this repo.

**Comment by gmarek** [comment]
@eparis - I think I have, as the 'merge pull request' button is green.

**Comment by eparis** [comment]
ok, found what @timothysc . Added Tim there. killed the team I made. We should have some consistency...

**Comment by timothysc** [comment]
Yes, I can merge now... thx @eparis ! 

@gmarek do we want to bot this repo?

File: RELEASE.md (hunk 1 above)
Comment (timothysc) [inline RELEASE.md]: k.

**Comment by timothysc** [comment]
lgtm

**Comment by eparis** [comment]
`do we want to bot this repo`?  The answer is yes. Consistency across kube is a good thing...

**Comment by gmarek** [comment]
Yes. We certainly need CLA bot and a basic build/unit test one. But I'm not sure yet if we need anything else.

**Pull request merged**